doc/nrf/protocols/matter/end_product/certification.rst

```diff
@@ -339,6 +335,37 @@ This variant is meant for Matter components that use Matter software that was al
 
    Matter certification by similarity overview
 
+Matter recertification process variants
+***************************************
+
+The following variants are meant for devices that are already certified.
+
+FastTrack Recertification program
+=================================
+
+Matter FastTrack Recertification is a program for recertification of previously certified Matter products.
+With FastTrack, you can test updated products and obtain a Matter certification of the result.
+The conformance of every certified Matter product is initially validated by an Alliance-approved Authorized Test Lab (ATL), but with FastTrack, you can validate the conformance of software updates to your certified Matter product without returning to the ATL.
+However, you need to send the updated firmware to the CSA Interoperability Lab.
+
+For more details, read a dedicated document in the `Matter Resource Kit`_.
+
+Rapid Recertification program
+=============================
+
+.. note::
+   The Rapid Recertification program is currently experimental.
+   You can read more about it in a dedicated document in the `Matter Resource Kit`_.
+
+The Rapid Recertification program lets you perform the product testing using the services of a qualified CSA Member, even your own organization, instead of an ATL.
+This allows for a potentially more streamlined recertification process when you want to certify.
+The first certification must still be done at an ATL.
+
+.. figure:: images/matter_device_certification_process_rr.svg
```

Comment:
Would it be possible to have similar pictur for Fast-Track? I guess the main difference would be in removing of ATL and IoP testing.

@doublemis1 do you have any idea how to present it nicely? We could discuss it F2F

Reply:
I think that based on current picture, we can make following changes:
![Screenshot from 2025-05-23 09-45-52](https://github.com/user-attachments/assets/d1afd4a3-3610-4b1d-8a5e-984dc4397233)